documents/eval_reports/reproduction.md

```
@@ -10,5 +10,5 @@
 | safety_car | auc_pr_test | 0.0723 | 0.0723 | reproduced | |delta| 0.0000 vs tol 0.01 |
 | undercut | auc_pr_test | 0.6739 | 0.6739 | reproduced | |delta| 0.0000 vs tol 0.01 |
 | pit_duration | p50_mae_test_s | 0.487 | 0.4893 | reproduced | n=252; |delta| 0.0023 vs tol 0.01 |
-| pace | mae_test_s | 0.4104 | - | pending | laptime holdout feature build not wired this phase |
-| tire_degradation | mae_test_s | 0.7078 | - | pending | TCN MC forward pass not run this phase (see calibration MC-sigma) |
+| pace | mae_test_s | 0.4104 | 0.4104 | reproduced | n=21247; |delta| 0.0000 vs tol 0.01 |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 6; Actual: 8; Too many cells, extra data will be missing

(MD056, table-column-count)